lib/src/swipable_stack.dart

```diff
@@ -294,6 +323,13 @@ class _SwipableStackState extends State<SwipableStack>
         setState(() {});
       });
     }
+
+    if (widget.dragStartBehavior == DragStartBehavior.down) {
+      _dragStartAnimation.curve = widget.dragStartCurve;
+      _dragStartController.duration = widget.dragStartDuration;
+    } else {
+      _dragStartController.duration = Duration.zero;
+    }
```

Comment:
I feel that this conditional flow is not necessary. It seems to more natural not to invoke `_dragStartController.forward(from: 0);` instead 🤔